tests/test_github.py

```diff
@@ -127,8 +127,44 @@ def test_actions(self, mock_request):
         self.assertEqual(a.project_id, project.id)
 
 
+    @patch('actionSHARK.github.GitHub.paginating', side_effect=mock_return)
+    def test_update_without_change(self, mock_request):
+        cfg = Namespace(tracking_url='https://localhost/repos/smartshark/test/actions')
+        project = Project.objects.get(name='test')
+        gp1 = GitHub(tracking_url=cfg.tracking_url, token='', owner='apache', repo='commons-rdf', project=project)
+        gp1.run()
+        gp2 = GitHub(tracking_url=cfg.tracking_url, token='', owner='apache', repo='commons-rdf', project=project)
+        gp2.run()
+        w = ActionWorkflow.objects.get(external_id='19196438')
+        r = ActionRun.objects.filter(external_id='7206977332')
+        j = ActionJob.objects.filter(external_id='19632879334')
+        a = RunArtifact.objects.filter(external_id='1111849070')
+
+        self.assertEqual(len(w.ci_system_ids), 2)
+        self.assertEqual(len(r), 1)
+        self.assertEqual(len(j), 1)
+        self.assertEqual(len(a), 1)
 
 
+    @patch('actionSHARK.github.GitHub.paginating', side_effect=mock_return)
+    def test_update_with_changes(self, mock_request):
```

Comment:
If updated data is created it should at least also test whether the list of ci_systems is updated accordingly to the new run, not just that the length of the collections increased.

Reply:
Done